### DISCUSSION TITLE
Static pages + the ability to leave comments

### DISCUSSION THREAD

**Discussion by oim37** [wintercms/winter#925] [Q&A]
Hello.

I want to add a field and a button so that visitors can leave their opinion about the text, for the text I use the module static pages.
Maybe someone has a ready-made solution to add this functionality, the text contains information on setting up services and users, having read it, could add their comments, share their opinion. A field and a button to add a comment, while to add you need to enter an email and a confirmation will be sent to it so that spam does not fall.

Is there any add-on or module ?

**Comment by LukeTowers** [comment]
@oim37 you could probably do this by adding an ajax handler to the PHP code section of your layout that handles your static pages.

**Comment by oim37** [comment]
I found plugin https://github.com/dimsog/wn-comments-plugin but it not working for me, wait answer from developer https://github.com/dimsog/wn-comments-plugin/issues/9

Based on the description, this is what I need.

**Comment by oim37** [comment]
thanks for the help
